### PR TITLE
UXG-1275 adds activate agents guided setup page

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1275

### CODE REVIEW

File: src/vue/backend/src/store/modules/listingsSettings/actions.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+const saveListingsSettings = ({ commit }, payload) => {
```
Comment (sgreenl5) [inline src/vue/backend/src/store/modules/listingsSettings/actions.js:1]: This seems like it is a duplicate of saveGeneralListingsSettings

File: src/vue/backend/src/views/guided-setup/Agents.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        ...mapState({
+            enabled: state => state.agentSettings.enabled,
+            guidedSetupSteps: state => state.progressStepper.guidedSetupSteps
+        })
+    },
+    methods: {
+        ...mapActions({
+            setItem: 'agentSettings/setItem',
+            progressStepperUpdate: 'progressStepper/progressStepperUpdate',
+            saveAgentSettings: 'general/saveAgentSettings'
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Agents.vue:61]: This should be in `agentSettings`, not `general`

File: src/vue/backend/src/views/guided-setup/Agents.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        async goContinue () {
+            await this.saveAgentSettings()
+            this.saveAction()
+            this.$router.push({ path: this.continuePath })
+        }
+    },
+    created () {
+        this.module = 'agentSettings'
+        this.cardTitle = 'Activate IMPress Agents'
+        this.activateLabel = 'Activate'
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Agents.vue:72]: We want to use `Enable` now

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

File: src/vue/backend/src/store/modules/listingsSettings/actions.js (hunk 1 above)
Comment (nicmarson) [reply src/vue/backend/src/store/modules/listingsSettings/actions.js:1]: I think there are four listings pages: Listings.vue, ListingsGeneral.vue, ListingsIdx.vue, and ListingsAdvanced.vue.
Listings.vue has the Enable switch.

Comment (sgreenl5) [reply src/vue/backend/src/store/modules/listingsSettings/actions.js:1]: If you are making one just to ping the enable endpoint, then the name should be more specific to that. Something like `enable____`

File: src/vue/backend/src/store/modules/agentSettings/actions.js [hunk 4]
```diff
@@ -1,2 +1,8 @@
+const enableAgentSettings = ({ commit }, payload) => {
```
Comment (sgreenl5) [inline src/vue/backend/src/store/modules/agentSettings/actions.js:1]: Enabled agent settings is not exactly what this is doing, instead it in enabling the agents plugin. Try `enableImpressAgents`

File: src/vue/backend/src/store/modules/listingsSettings/actions.js [hunk 5]
```diff
@@ -1,3 +1,7 @@
+const enableListingsSettings = ({ commit }, payload) => {
```
Comment (sgreenl5) [inline src/vue/backend/src/store/modules/listingsSettings/actions.js:1]: Same as comment above about agents, this is to enable the listings plugin

File: src/vue/backend/src/views/guided-setup/Agents.vue [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    mixins: [
+        guidedSetupMixin,
+        pageGuard
+    ],
+    components: {
+        GuidedSetupContentCard
+    },
+    computed: {
+        ...mapState({
+            enabled: state => state.agentSettings.enabled,
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Agents.vue:53]: Are you using `enabled` here?

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

File: src/vue/backend/src/views/guided-setup/Agents.vue (hunk 6 above)
Comment (nicmarson) [reply src/vue/backend/src/views/guided-setup/Agents.vue:53]: Yeah, it's the activate plugin slider, and it's one of the store settings.

Comment (sgreenl5) [reply src/vue/backend/src/views/guided-setup/Agents.vue:53]: The activate slider plugin is using `localStateValues`, not the state directly

**Review by sgreenl5 — APPROVED** [review]

**Review by ErikIDX — APPROVED** [review]

**Pull request merged**